src/main/groovy/org/transmartproject/core/biomarker/BioMarkerResult.groovy

```diff
@@ -6,4 +6,4 @@ package org.transmartproject.core.biomarker
  *
  * In general, the iterator can be fetched only once.
  */
-interface BioMarkerResult extends Closeable, Iterable<BioMarker> {}
+interface BioMarkerResult extends Iterable<BioMarker> {}
```

Comment:
This change is problematic, from what I see in core-db. You removed the `Closeable` interface from `BioMarkerResult`, but looking at the implementation:

```
class BioMarkerScrollableResultsWrappingIterable extends SimpleScrollableResultsWrappingIterable<BioMarker> implements BioMarkerResult {

    BioMarkerScrollableResultsWrappingIterable(ScrollableResults scrollableResults) {
        super(scrollableResults)
    }
}
```

and the superclass...:

```
class SimpleScrollableResultsWrappingIterable<T> extends AbstractOneTimeCallIterable<T> {

    protected final SimpleScrollableResultsIterator scrollableResultsIterator

    SimpleScrollableResultsWrappingIterable(ScrollableResults scrollableResults) {
        this.scrollableResultsIterator = new SimpleScrollableResultsIterator(scrollableResults)
    }

    @Override
    protected Iterator getIterator() {
        scrollableResultsIterator
    }

}
```

```class SimpleScrollableResultsIterator<T> implements Iterator<T>, Closeable {```

Three classes later we have a closeable object that's not exposed as such to the client, and therefore that the client can't close.